srml/council/src/seats.rs

```diff
@@ -565,6 +679,45 @@ impl<T: Trait> Module<T> {
 		<VoteCount<T>>::put(Self::vote_index() + 1);
 		Ok(())
 	}
+
+	/// Return a concatenated vector over all voter sets.
+	fn all_voters() -> Vec<(T::AccountId, BalanceOf<T>)> {
```

Comment:
This function is so, SO, dangerously wrong.

NEVER EVER `::enumerate()` in the runtime over an unbounded set.

Reply:
Okay so the places that I use the bad function: 

https://github.com/paritytech/substrate/blob/d6ceac40d9f21e3507b9dbcd3a883ab245b76c30/srml/council/src/seats.rs#L155-L159

Not really needed, can change it.

https://github.com/paritytech/substrate/blob/d6ceac40d9f21e3507b9dbcd3a883ab245b76c30/srml/council/src/seats.rs#L202-L207

See the TODO.

https://github.com/paritytech/substrate/blob/d6ceac40d9f21e3507b9dbcd3a883ab245b76c30/srml/council/src/seats.rs#L275-L287

This one cannot be avoided. I can still __not concatenate it__ but at the end of the day, we are reading all voters, since we need all of them.

https://github.com/paritytech/substrate/blob/d6ceac40d9f21e3507b9dbcd3a883ab245b76c30/srml/council/src/seats.rs#L629-L638

Same as the above. This is the consequence of the fact that now we save some sort of a `last_win_index`.